tests/bin/prices.test

```diff
@@ -1,7 +1,7 @@
 # Test prices addon
 
 # by default only market prices are reported
-../../bin/hledger-prices -f-
+../../bin/hledger-prices.hs -f-
```

Comment:
5a7f74c6ffbe7cd39404302aa5ab37ec2e1629ed made all the bin/* tests use compiled executables, in quest of speedier tests. It probably would be best to discuss this separately, and not to do it for just one file.